docker/latest-node-compose.yml

```diff
@@ -5,7 +5,6 @@ services:
       - "9944:9944"
     command:
       [
-        "storage-hub-node",
```

Comment:
How's this working without the command of the binary?

Reply:
Docker image entrypoint should point directly to the binary so you dont need to specify cmd explicitly.

This compose file wont work until PR is merged though as latest images (from `main` branch) aren't using the new entry point.